manifests/ssl.pp

```diff
@@ -2,10 +2,24 @@
 #
 # Apache resources specific for SSL
 #
-class apache::ssl {
+class apache::ssl (
+  $ssl_port            = params_lookup( 'ssl_port' ),
+  $ssl_source          = params_lookup( 'ssl_source' ),
```

Comment:
Agree, better to leave the params in the ssl class.
Still looking at the params_lookup function (https://github.com/example42/puppi/blob/master/lib/puppet/parser/functions/params_lookup.rb), I wonder if the lookup names are what we expect. For example I'm not sure the default values are searched in apache::params